hotwrap.go

```diff
@@ -51,14 +51,29 @@ func withError(execName string, execArgs []string) fdk.HandlerFunc {
 func runExec(ctx context.Context, execCMDwithArgs string, in io.Reader, out io.Writer) error {
 	log.Println(execCMDwithArgs)
 	fctx := fdk.GetContext(ctx)
+	hdr := fctx.Header()
```

Comment:
you'd need to try to check whether you can create HTTP context, if you can do that you'd need to take headers from http context, if not - take headers from invoke context.